pom.xml

```diff
@@ -26,11 +26,8 @@
         <skipUnitTests>false</skipUnitTests>
     
         <jakartaee-api.version>8.0.0</jakartaee-api.version>
-        <!-- Make this correspond to the version used in Payara: 5.201 uses (patched) 2.3.14 -->
-        <mojarra.version>2.3.14</mojarra.version>
+        <payara.version>5.2020.2</payara.version>
```

Comment:
Does this mean that this new "payara.version" property needs to be bumped every time we want to move to a new version of Payara?

Reply:
It should be moved. This is not a must, but a strong should.

This hasn't happened in the past with GF 4.1, as that appserver hadn't any updates. It won't be much of a hassle to do this and it makes testing with newer appserver versions easier.

Reply:
By the way, as the article outlined, too: you would do this with the Wildfly/Jetty/Liberty BOM when using those appservers.

You might think about creating different maven profiles when multi appserver compatibility becomes a feature.

Reply:
Ok. Maybe after we've done a few of these Payara upgrades (pull request #7036 was the last one), we'll get into a nice pattern of which files need to be touched (similar to how we know which files to touch when making a release, and even document them). So heads up here to @donsizemore (who made the last bump) that a new line is being added.